src/cli/common.js

```diff
@@ -22,8 +23,12 @@ export function sourcecredDirectory(): string {
   return env != null ? env : defaultSourcecredDirectory();
 }
 
-export function githubToken(): string | null {
-  return NullUtil.orElse(process.env.SOURCECRED_GITHUB_TOKEN, null);
+export function githubToken(): GithubToken | null {
+  const envToken = NullUtil.orElse(process.env.SOURCECRED_GITHUB_TOKEN, null);
+  if (envToken == null || !envToken.length) {
+    return null;
+  }
```

Comment:
The usage of `NullUtil.orElse` here seems like a coersion from `undefined` to `null`.
So we could skip this with a falsy check.

```js
const envToken = process.env.SOURCECRED_GITHUB_TOKEN;
if (!envToken) {
  return null;
}
```

Reply:
Changed per your request.